db/sql/migrations/runtime.sql

```diff
@@ -19,11 +19,12 @@
   PARAMETERS: None
 */
 
-UPDATE "peers" SET "state" = 1, "clock" = NULL WHERE "state" != 0;
-
 -- Overwrite unconfirmed tables with state from confirmed tables
-DELETE FROM mem_accounts2u_delegates;
-INSERT INTO mem_accounts2u_delegates ("accountId", "dependentId") SELECT "accountId", "dependentId" FROM mem_accounts2delegates;
+DROP TABLE mem_accounts2u_delegates;
+CREATE TABLE mem_accounts2u_delegates AS TABLE mem_accounts2delegates;
+ALTER TABLE mem_accounts2u_delegates ADD CONSTRAINT "mem_accounts2u_delegates_accountId_fkey" FOREIGN KEY ("accountId") REFERENCES mem_accounts(address) ON DELETE CASCADE;
+CREATE INDEX "mem_accounts2u_delegates_accountId" ON mem_accounts2u_delegates("accountId");
 
 DELETE FROM mem_accounts2u_multisignatures;
 INSERT INTO mem_accounts2u_multisignatures ("accountId", "dependentId") SELECT "accountId", "dependentId" FROM mem_accounts2multisignatures;
```

Comment:
Please make the change also for this query.

Reply:
I will though impact was less as it was changing a small number of rows but it makes sense for both queries to look the same

Reply:
Yep, just for consistency. Also, there are some unit tests that are not passing after the change.

Reply:
Yeah checking the unit tests but I think they were not passing without the change; Just merged develop back in and will check again.

Reply:
It's because of the change in this PR:
```
16:50:10        MigrationsRepository
16:50:10          applyRuntime()
16:50:10            should set peers to disconnected mode and clock to NULL for non-banned peers:
16:50:10 
16:50:10       [31mAssertionError: expected 0 to deeply equal 1[0m
16:50:10       [32m+ expected[0m [31m- actual[0m
16:50:10 
16:50:10       [31m-0[0m
16:50:10       [32m+1[0m
16:50:10       [0m[90m
16:50:10     at Context.<anonymous> (test/unit/db/repos/migrations.js:170:38)
16:50:10     at Generator.next (<anonymous>:null:null)
16:50:10     at onFulfilled (node_modules/co/index.js:65:19)
16:50:10     at tryCatcher (node_modules/bluebird/js/release/util.js:16:23)
16:50:10     at Promise._settlePromiseFromHandler (node_modules/bluebird/js/release/promise.js:512:31)
16:50:10     at Promise._settlePromise (node_modules/bluebird/js/release/promise.js:569:18)
16:50:10     at Promise._settlePromise0 (node_modules/bluebird/js/release/promise.js:614:10)
16:50:10     at Promise._settlePromises (node_modules/bluebird/js/release/promise.js:693:18)
16:50:10     at Async._drainQueue (node_modules/bluebird/js/release/async.js:133:16)
16:50:10     at Async._drainQueues (node_modules/bluebird/js/release/async.js:143:10)
16:50:10     at Immediate.Async.drainQueues [as _onImmediate] (node_modules/bluebird/js/release/async.js:17:14)
16:50:10   
```